modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -62,7 +62,7 @@ const Diagram = ({
 	});
 
 	useEffect(() => {
-		if (!showTooltip.tooltip && showTooltip.details.id) {
+		if (!showTooltip.tooltip) {
 			const myNewState = cPins.map((element) => {
```

Comment:
Who is `my` in this `myNewState`? Should be something that describes what that state is exactly, in my opinion.